.rubocop.yml

```diff
@@ -1,6 +1,15 @@
 # Enable this when the VSC rubocop extension supports rails
 #inherit_from: ./config/rails.rubocop.yml
 
+AllCops:
+  # Exclude automatically generated rails files for easier app:update's etc
+  Exclude:
+    - bin/*
+    - db/schema.rb
+    - config/boot.rb
+    - config/environment.rb
+    - config/environments/*
```

Comment:
So I made the excludes a bit more specific as discussed in https://github.com/sydevs/WeMeditate/pull/41#pullrequestreview-315051265

We could go even further and enforce styling on all these files except `schema.rb` if we want however I would like to defer that to a separate pull request where we can also run the rubocop autoformatter on all the files.

Reply:
Alright, sounds reasonable.